cypress/e2e/tests/pages/explorer/api/api-services.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { APIServicesPagePo } from '@/cypress/e2e/po/pages/explorer/api-services.po';
+
+describe('Cluster Explorer', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  describe('API: APIServices', () => {
+    let apiServicesPage: APIServicesPagePo;
+
+    beforeEach(() => {
+      apiServicesPage = new APIServicesPagePo('local');
+      apiServicesPage.goTo();
+    });
+
+    it('Should have a title', () => {
+      cy.get('h1').should('contain', 'APIServices');
+    });
+
+    it('Should be able to use shift+j to select corre', () => {
+      cy.get('.main-layout').click();
+
+      const count = 4;
+
+      cy.keyboardControls({ shiftKey: true, key: 'j' }, 4);
+
+      cy.get('#selected-count').should('contain', `${ count } selected`);
```

Comment:
You probably want to use `BaseResourceList` for this and below.

I can't remember which part has the selected text, but there should be a new getter in that po that returns the text. That po is then accessed via the instance of BaseResourceList.

For the checked box count that should be a new getter on the sortable table po, accessed here by the instance of BaseResourceList.

This way when someone else needs to use similar functionality it's there ready, and if that functionality changes we only need to update in that single location